.env-sample

```diff
@@ -1,4 +1,7 @@
 //N.B: When writting the below credentials ommit the curl braces.
```

Comment:
Comments should start with `#` and not `//`

Reply:
ok. Solved in the latest push